project/data/prod/data.yaml

```diff
@@ -158,7 +158,7 @@
 06-04: ~
 06-05: ~
 06-06: 
-    - 'Patrick Rothfuss, Autor der wohl besten Fantasybücher, Träger eines wunderhübschen Bartes, und Gründers der Wohltätigkeitsorganisation "Worldbuilders" wird heute {1973}. Come on, Pat, we are still waiting for book three!!!.'
+    - 'Patrick Rothfuss, Autor der wohl besten Fantasybücher, Träger eines wunderhübschen Bartes und Gründer der Wohltätigkeitsorganisation "Worldbuilders" wird heute {1973}. Come on, Pat, we are still waiting for book three!!!.'
```

Comment:
[...] Come on Pat, we are still waiting [...]

Ohne das jetzt mit Grammatikregeln begründen zu können, würde ich sagen dass man das Komma vor dem Namen weglassen sollte. Liest sich irgendwie viel natürlicher.